Dockerfile

```diff
@@ -48,20 +52,34 @@ ENV CATALINA_OPTS="\$EXTRA_JAVA_OPTS \
 
 # init
 RUN apt update && apt -y upgrade && \
-    apt install -y openssl zip gdal-bin wget curl openjdk-11-jdk grass-dev libpq-dev make g++ checkinstall && \
+    apt install -y openssl zip gdal-bin wget curl openjdk-11-jdk libpq-dev \
+    devscripts make g++ checkinstall && \
     rm -rf $CATALINA_HOME/webapps/*
 
+RUN echo "deb-src https://ppa.launchpadcontent.net/ubuntugis/ubuntugis-experimental/ubuntu jammy main" >> /etc/apt/sources.list
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 089EBE08314DF160
+RUN apt update
+RUN apt-get source grass
+RUN apt build-dep grass -y
+WORKDIR /grass-${GRASS_VERSION_FULL}
+RUN debuild -b -uc -us
+RUN echo /usr/lib/grass${GRASS_VERSION}/lib > /etc/ld.so.conf.d/grass.conf && ldconfig
+
+# install GRASS GIS packages for GDAL-GRASS driver compilation
+RUN dpkg -i /grass-core_${GRASS_VERSION_FULL}*_amd64.deb \
+    /grass-dev_${GRASS_VERSION_FULL}*_amd64.deb \
+    /grass-doc_${GRASS_VERSION_FULL}*_all.deb
```

Comment:
You can try to just install `grass-core` that's the only one necessary for running the geoserver store

Reply:
Sorry, wrong place

Reply:
Yes, changed (in the right place) in 4b7fd73